src/FunderMaps.Core/Extensions/ExceptionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
```

Comment:
The necessity for this extension is lost on my. Extensions add functionality to otherwise inaccessible (closed) objects. Both the extension and the extended class are captured within the same assemble.